cluster/src/assembly/resources/conf/iotdb-cluster.properties

```diff
@@ -58,7 +58,13 @@ max_concurrent_client_num=10000
 default_replica_num=2
 
 # connection time out (ms) among raft nodes
-connection_time_out_ms=20000
+connection_timeout_ms=20000
+
+# write operation timeout threshold (ms)
+write_operation_timeout_ms=30000
+
+# read operation timeout threshold (ms)
+read_operation_timeout_ms=30000
```

Comment:
Add comments explaining that the configs are only for internal communications, not for the whole operation.

Reply:
Thanks for  your advice. I will add these comments.